src/saving/SaveManagerGUI.cs

```diff
@@ -102,14 +107,16 @@ public override void _Process(float delta)
             Math.Round((float)info.diskSpace / Constants.MEBIBYTE, 2).ToString(CultureInfo.CurrentCulture) + " MiB";
 
         refreshing = false;
+
+        UpdateSelectedCount();
```

Comment:
I would like these two lines to be above the `refreshing = false` line as conceptually I imagine that to be like the end of this method, marking things as done.